contact-builder/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM rust
```

Comment:
Need to pin a specific rust version, such as `rust:1.44.1`
 https://hub.docker.com/_/rust